mne/commands/mne_flash_bem_model.py

```diff
@@ -23,8 +23,10 @@
 import os
 
 import mne
+from mne.utils import deprecated
```

Comment:
Not really, I didn't noticed I changed the file permissions. How do I put them back as they were before? Git doesn't seems to be updating them.